composer.json

```diff
@@ -20,8 +20,7 @@
         "mongodb/mongodb": "^1.1",
```

Comment:
Looks like adding `composer.lock` affected MongoDB. Is there something to do, or should we not include it here?